web/public/js/directives/cd-rounded-icon/index.js

```diff
@@ -0,0 +1,24 @@
+// Font awesome rounded icon directive
+// To use call `div cd-rounded-icon src="icon-name"></div>` you just need the name of the icon without the fa
+;(function() {
+  'use strict';
+
+function cdRoundedIcon(){
+    return {
+      scope: {
+        src: '@?',
+      },
+      restrict: 'EA',
+      templateUrl: '/directives/tpl/cd-rounded-icon',
+      controller: ['$scope', function ($scope) {
```

Comment:
This controller doesn't seem necessary. You're setting the scope through the scope object above via an attribute. It should work fine without this controller.